conf/conf_example.xml

```diff
@@ -241,39 +153,6 @@
    settings_modules="modules/video_rtp_stream.xml modules/geo_mag.xml modules/air_data.xml modules/gps_ubx_ucenter.xml modules/ins_extended.xml modules/ahrs_int_cmpl_quat.xml modules/stabilization_int_quat.xml modules/nav_basic_rotorcraft.xml modules/guidance_rotorcraft.xml modules/gps.xml modules/imu_common.xml"
    gui_color="red"
   />
-  <aircraft
-   name="ardrone2_flightplan_guided_gazebo"
-   ac_id="35"
-   airframe="airframes/examples/ardrone2_gazebo.xml"
-   radio="radios/dummy.xml"
-   telemetry="telemetry/default_rotorcraft.xml"
-   flight_plan="flight_plans/rotorcraft_guided_flightplan.xml"
-   settings="settings/rotorcraft_basic.xml settings/nps.xml"
-   settings_modules="modules/video_rtp_stream.xml modules/cv_colorfilter.xml modules/geo_mag.xml modules/air_data.xml modules/gps_ubx_ucenter.xml modules/ins_extended.xml modules/ahrs_int_cmpl_quat.xml modules/stabilization_int_quat.xml modules/nav_basic_rotorcraft.xml modules/guidance_rotorcraft.xml modules/gps.xml modules/imu_common.xml"
-   gui_color="red"
-  />
-  <aircraft
-   name="ardrone2_gazebo"
```

Comment:
I think one of the two of these should stay as there seems to be no other reference to this airframe in a different conf.

Reply:
Yes, but gazebo is not loaded so it will fail when used for test. I can eventually add it back, but a solution that would allow the CI server to work would be nice. How long does it take to download gazebo ?

Reply:
Oh yea, I forgot about that. 

I just tried on my pc, took 30s to download and install (apt install line) and 4s to update the models submodule.

I leave it to you to include it in the CI tests but it seems like it could be done (till the number of models grow out of control ;P).

Reply:
which version of gazebo are you using ? gazebo7 ? do you need something else beside `apt install gazeboX` ?

Reply:
I secretly used apt install gazebo9 as that was included in the current apt repo. If you need 7 you will have to first add the repo, e.g. http://gazebosim.org/tutorials?tut=install_ubuntu&ver=7.0

Any input here @tomvand

Reply:
Gazebo 7 is the only version that works correctly right now. I'm running 8.0 on my pc, but this version is no longer available and 8.3 has breaking performance issues. Gazebo 9 is not compatible because of API changes, I plan to fix this for next year's MAV course (around february).

As for the models: all essential parts are already included with paparazzi (ardrone, bebop models and an empty world). The tudelft_gazebo_models submodule is not required to test the simulator.

Reply:
I tried v9 (the long term support version...) was 50% though adjustments for v9, time up...  v9 support appreciated and for TU Delft 2019 course probably the target version